### PR TITLE
SCP-4772 Comparison of Marlowe chainindex to Cardano dbsync

### PR DESCRIPTION
**Pull request by bwbush**
SQL scripts for comparing the database for `marlowe-chainindexer` against `cardano-db-sync`. See http://testing.marlowe.run/database/ for example output and log files.

Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: marlowe-test/chain/upload.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+gsutil -m -h 'Cache-Control:public,max-age=600' rsync -x ".*\.csv$" out gs://testing.marlowe.run/database/
+
```
Comment (jhbertra) [inline marlowe-test/chain/upload.sh:4]: What's this about?

**Review by jhbertra — APPROVED** [review]
LGTM, just a couple questions

File: marlowe-test/chain/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+out/
```
[the remark this answers is not in the document]
Comment (bwbush) [reply marlowe-test/chain/.gitignore:1]: Sure. I had no idea we were avoiding nested `.gitignore`s. We should probably documentation that somewhere like in the CONTRIBUTING doc.

File: marlowe-test/chain/upload.sh (hunk 1 above)
Comment (bwbush) [reply marlowe-test/chain/upload.sh:4]: Uploading test results to https://testing.marlowe.run/database/. This is temporary until we have automated overnight runs.

**Pull request merged**